src/server/adminBro.ts

```diff
@@ -405,6 +408,105 @@ const getAdminBroInstance = () => {
           },
         },
       },
+      {
+        resource: Token,
+        options: {
+          properties: {
+            networkId: {
+              isVisible: true,
+              availableValues: [
+                { value: NETWORK_IDS.MAIN_NET, label: 'MAINNET' },
+                { value: NETWORK_IDS.ROPSTEN, label: 'ROPSTEN' },
+                { value: NETWORK_IDS.XDAI, label: 'XDAI' },
+                { value: NETWORK_IDS.BSC, label: 'BSC' },
+              ],
+            },
+            symbol: { isVisible: true },
+            name: { isVisible: true },
+            isGivbackEligible: { isVisible: true },
+            address: { isVisible: true },
+            mainnetAddress: {
+              isVisible: {
+                show: true,
+                edit: true,
+                new: true,
+                list: false,
+                filter: true,
+              },
+            },
+            decimals: { isVisible: true },
+            organizations: {
+              isVisible: {
+                show: true,
+                edit: true,
+                new: true,
+                list: true,
+              },
+              components: {
+                show: AdminBro.bundle('./components/ListOrganizationsNames'),
+                list: AdminBro.bundle('./components/ListOrganizationsNames'),
+              },
+              availableValues: [
```

Comment:
Honestly found no other way to do it.
Tried the { isMulti: true } but adminbro ignores the array of values.

Reply:
I had this problem before